website/docs/reference/resource-configs/duckdb-configs.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "DuckDB configurations"
+id: "duckdb-configs"
+---
+
+
+## Profile
```

Comment:
Profile information should belong in https://docs.getdbt.com/docs/core/connect-data-platform/duckdb-setup since dbt Cloud users don't have to create their own profiles.yml files